crypto/evp/ctrl_params_translate.c

```diff
@@ -1344,7 +1344,7 @@ static int fix_rsa_pss_saltlen(enum state state,
                 break;
         }
         if (i == OSSL_NELEM(str_value_map)) {
-            BIO_snprintf(ctx->name_buf, 5, "%d", ctx->p1);
+            BIO_snprintf(ctx->name_buf, sizeof(ctx->name_buf), "%d", ctx->p1);
```

Comment:
I think we will have more issues if OSSL_MAX_NAME_SIZE become less than 5 :)

Reply:
The number of digits in ctx->p1 has to equal or exceed OSSL_MAX_NAME_SIZE.
That is: _⌈log₁₀(ctx->p1)⌉ ≥ OSSL_MAX_NAME_SIZE_